tf2_kdl/CMakeLists.txt

```diff
@@ -28,26 +20,26 @@ install(PROGRAMS scripts/test.py
          DESTINATION lib/${PROJECT_NAME}
 )
 
-
-# if(BUILD_TESTING)
-#   # find_package(ament_cmake REQUIRED COMPONENTS rostest tf2 tf2_ros tf2_msgs)
-#   add_executable(test_kdl EXCLUDE_FROM_ALL test/test_tf2_kdl.cpp)
-#   find_package (Threads)
-#   target_link_libraries(test_kdl ${rclpy_LIBRARIES}
-#                                 ${rmw_implementation_LIBRARIES}
-#                                 ${GTEST_LIBRARIES}
-#                                 ${orocos_kdl_LIBRARIES}
-#                                 ${GTEST_LIBRARIES}
-#                                 ${CMAKE_THREAD_LIBS_INIT}
-# )
-#
-#   ament_add_gtest(${CMAKE_CURRENT_SOURCE_DIR}/test/test.launch)
-#   ament_add_gtest(${CMAKE_CURRENT_SOURCE_DIR}/test/test_python.launch)
-#
-#   if(TARGET tests)
-#     add_dependencies(tests test_kdl)
-#   endif()
-# endif()
+if(BUILD_TESTING)
+	find_package(ament_cmake_gtest REQUIRED)
+
+  ament_add_gtest(test_kdl test/test_tf2_kdl.cpp)
+
+	target_include_directories(test_kdl PUBLIC
```

Comment:
The approach you follow here is different from what's been used at https://github.com/ros2/geometry2/pull/90/files#diff-61737c5a9ef69ef2118eafbc59732b6aR45